GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -1254,5 +1254,25 @@ private void SetBranchesPushCheckboxesState(Func<DataGridViewRow, bool> willPush
                 pushCheckBox.Value = willPush(row);
             }
         }
+
+        internal TestAccessor GetTestAccessor() => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly FormPush _form;
+
+            public TestAccessor(FormPush form)
+            {
+                _form = form;
+            }
+
+            public CheckBox ForcePushBranches() => _form.ForcePushTags;
+
+            public CheckBox ckForceWithLease() => _form.ckForceWithLease;
+
+            public CheckBox ForcePushTags() => _form.ForcePushTags;
```

Comment:
No need for methods, use readonly properties:
```suggestion
            public CheckBox ForcePushTags => _form.ForcePushTags;

            public CheckBox ckForceWithLease => _form.ckForceWithLease;
```

Why is `_form.ForcePushTags` referenced twice under different names?

Reply:
@RussKie 😦 good catch, this is an error.  It should be `_form.ForcePushBranches`...

Reply:
Fixed!